app/src/main/java/com/example/fittracker/MainActivityView.java

```diff
@@ -0,0 +1,42 @@
+package com.example.fittracker;
+
+import android.app.Activity;
+import android.widget.EditText;
+import android.widget.Toast;
+
+public class MainActivityView implements LogInContract.View{
+    private EditText mail;
+    private EditText password;
+    private Activity actividad;
+    public MainActivityView(EditText mail, EditText password,Activity actividad){
```

Comment:
don't pass these parameters here. You can access those views with butterknife

Reply:
Done